doc/physical_validation.rst

```diff
@@ -16,7 +16,7 @@ physical_validation\.ensemble module
 .. automodule:: physical_validation.ensemble
     :members:
 
-physical_validation\.integratorconvergence module
+physical_validation\.integrator.convergence module
```

Comment:
Actually, the module is `physical_validation\.integrator`, convergence is the function. Also, please adjust the length of the line below accordingly, I think to remember that it can trigger warnings or errors when deploying the documentation otherwise.